uni/test/test_widget.dart

```diff
@@ -11,6 +14,8 @@ Widget testableWidget(
 }) {
   return MultiProvider(
     providers: [
+      ChangeNotifierProvider(create: (context) => SessionProvider()),
```

Comment:
Won't this cause conflict if testing this providers via parameter? Why is this needed?

Reply:
It fails to initialize the providers to test, causing a test error